### PR TITLE
LPS-147892 buildLang

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by Tim-Cao** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by Tim-Cao** [comment]
Hi @victorg1991, the select category modal is empty.
Steps to reproduce.

1. Add a dynamic collection
2. Select the item type and item subtype
3. Expand the Filter
4. Select the Categories
5. Click Select

image

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/BrowserSidebar.js [hunk 1]
```diff
@@ -26,29 +26,35 @@ import SidebarPanelHeader from '../../../common/components/SidebarPanelHeader';
...
+};
+
 const TABS = [
 	{
 		component: <PageStructureSidebar />,
 		label: Liferay.Language.get('page-elements'),
 	},
 	{
 		component: <ContentsSidebar />,
+
```
Comment (sandrodw3) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/BrowserSidebar.js:41]: This space is affecting my TOC seriously 🥺

**Review by sandrodw3 — COMMENTED** [review]
LGTM! Just left a tiny comment :peepo-clown:

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**